src/figuresClasses.ts

```diff
@@ -1,11 +1,99 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
```

Comment:
create type Shape = 'triangle' | 'circle' | 'rectangle' and type Color = 'red' | 'green' | 'blue';
Now you can reusing it in other places.